### DISCUSSION TITLE
why unchecking "Deep learning based spectra" checkbox give more idetifications

### DISCUSSION THREAD

**Discussion by amrelguo** [vdemichev/DiaNN#702] [Q&A]
I searched DIA endogenous peptide data against constructed DDA spectral library with and without checking "Deep learning-based spectra" check box. 
the results showed that more identification can be obtained without ticing "Deep learning-based spectra" checkbox.  i wondering why this happened and if these unique identificaions can be considered as significant ones or not.
thanks in advance.

**Comment by vdemichev** [comment]
Yes, very often empirical spectra, RTs and IMs can be better than predicted, this is expected.

**Comment by amrelguo** [comment]
Thanks so  much for your quick response.

**Comment by amrelguo** [comment]
despite I didn't tick "deep learning based spectra" checkbox, the deep learning was done with less number of targets and decoys than those used when i ticked. what is the reason?

[report.log_NO_deep_learning.txt](https://github.com/vdemichev/DiaNN/files/11512711/report.log_NO_deep_learning.txt)
[report.log_with_deep_learning.txt](https://github.com/vdemichev/DiaNN/files/11512713/report.log_with_deep_learning.txt)
